scripts/preview/utils/ecs.ts

```diff
@@ -151,12 +151,17 @@ async function getRunningTask(slug: string): Promise<AWS.ECS.Task | undefined> {
     desiredStatus: 'RUNNING',
   };
 
-  const tasks = await Clients.ecs.listTasks(params).promise();
+  const tasks: any = await Clients.ecs.listTasks(params).promise();
```

Comment:
can we remove the `any`?

Reply:
You can find the correct type from looking into `listTasks`

Reply:
Done